Jetcaster/README.md

```diff
@@ -3,60 +3,34 @@
 # Jetcaster sample 🎙️
 
 Jetcaster is a sample podcast app, built with [Jetpack Compose][compose]. The goal of the sample is to
-showcase dynamic theming and full featured architecture.
+showcase building with Compose across multiple form factors (mobile, TV, and Wear) and full featured architecture.
 
 To try out this sample app, use the latest stable version
 of [Android Studio](https://developer.android.com/studio).
 You can clone this repository or import the
 project from Android Studio following the steps
 [here](https://developer.android.com/jetpack/compose/setup#sample).
 
-### Status: 🚧 In progress 🚧
-
-Jetcaster is still in the early stages of development, and as such only one screen has been created so far. However,
-most of the app's architecture has been implemented, as well as the data layer, and early stages of dynamic theming.
-
-
 ## Screenshots
 
-<img src="docs/screenshots.png"/>
+![readme_cover](https://github.com/android/compose-samples/assets/10263978/a58ab950-71aa-48e0-8bc7-85443a1b4f6b)
```

Comment:
reminder that the Wear one does not have the download function right now, I can upload some screenshots if you like?

Reply:
Good call out. I don't think we'll be adding that feature at all, yeah? If so, we can update the screenshot to reflect the actual implementation on Wear.